### ISSUE TITLE
Training with big image size fail

### ISSUE THREAD

**Issue by aungsanwin** [qqwweee/keras-yolo3#582]
Hi,

Thank for your complete reference on tensorflow, keras for YoloV3.
I am training using big image size with 3 additional classes.
my data 
testimage_1.jpg 1479,553,1850,1001,2 1007,1129,1121,1277,1 2245,1101,2617,1215,0 512,758,860,886,0

I always get val_loss error after 3 epoch and could not complete my training.

**Comment by syikmis** [comment]
How big are your Images? They should get resized by means of calling method get_random_data() in yolo3/utils.py

**Comment by aungsanwin** [comment]
Hi Syikmis,
Thank for your kind response.
I saw in train.py call the data_generator_wrapper in turn call the get_random_data. But it seems my val_loss error comes from ReduceLROnPlateau. Is there any reason i loss in 3 epoch.
